strawberryfields/configuration.py

```diff
@@ -158,6 +206,31 @@ def load_config_file(filepath):
     return config_from_file
 
 
+def get_api_section_safely(loaded_config, filepath):
+    """Gets the API section from the loaded configuration.
+
+    Args:
+        loaded_config (dict): the configuration that was loaded from the toml
+            file
+        filepath (str): path to the configuration file
+
+    Returns:
+        dict[str, Union[str, bool, int]]: the api section of the configuration
+
+    Raises:
+        ConfigurationError: if the api section was not defined in the
+            configuration
+    """
+    try:
+        return loaded_config["api"]
+    except KeyError:
+        log = create_logger(__name__)
+        log.error(
+            "The configuration from the %s file does not 'contain an \"api\" section.'", filepath
+        )
+        raise ConfigurationError()
```

Comment:
Should this have the `()`?

Reply:
Hmm perhaps not, works with me without it! :)